src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, int age) {
+        this.login = login;
```

Comment:
The constructor parameter 'age' should be of type 'Integer' instead of 'int' to match the 'age' field type.